src/views/Dashboard/Customer/Home/Emr/Edit.vue

```diff
@@ -528,6 +521,10 @@ export default {
       this.clearFile = true
       this.showModalPassword = true
```

Comment:
Do you still need to show the pop up ?

Reply:
oh sorry, popup is not use anymore, delete the code? @ciniiia